automatic/virtualbox_5.0.x/virtualbox_5.0.x.nuspec

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>virtualbox</id>
+    <title>VirtualBox</title>
+    <version>0.0</version>
+    <authors>Oracle Corporation</authors>
+    <owners>chocolatey, Rob Reynolds, Mike Dellanoce</owners>
+    <summary>VirtualBox is a general-purpose full virtualizer for x86 hardware, targeted at server, desktop and embedded use.</summary>
+<description>
+
+VirtualBox is a cross-platform virtualization application. It installs on existing Intel or AMD-based computers, whether they are running Windows, Mac, Linux or Solaris operating systems. It extends the capabilities of your existing computer so that it can run multiple operating systems (inside multiple virtual machines) at the same time.
+
```

Comment:
@pascalberger I can't seem to find a features list on their website, mind if I skip it.
If we add it, it should also be added to the original virtualbox package